indico/modules/events/papers/templates/emails/base.html

```diff
@@ -0,0 +1,40 @@
+{% extends 'emails/base.html' %}
+
+{% block header %}{% endblock %}
+
+{% block footer %}{% endblock %}
+
+{% block subject -%}
+    {% block subject_message %}Call for papers{% endblock %}
+    {%- block subject_name %}{% endblock %}
+{%- endblock %}
+
+{% block body %}
+    <div class="mail" style="width: 600px; font-family: Arial; border: 1px solid #dfdfdf; background-color: #fcfcfc; margin: auto auto 40px auto;">
+        <div class="header" style="background-color: #5D95EA; color: #eee; padding: 10px 20px;">
```

Comment:
Hexadecimal notation should be lowercase according to our set of sass-lint rules.